jcloud/parsers/update.py

```diff
@@ -0,0 +1,19 @@
+from .base import set_base_parser
+from pathlib import Path
+
+def set_update_parser(parser=None):
+    if not parser:
+        parser = set_base_parser()
+
+    parser.add_argument(
+        'flow_id',
+        help='The string ID of the flow to be updated',
+    )
```

Comment:
Let's use `flow` here instead of `flow_id` as that's followed for other commands.

Reply:
sure

Reply:
Done.